src/service/cia301/co_csdo.h

```diff
@@ -133,6 +133,17 @@ extern "C" {
 * PUBLIC TYPES
 ******************************************************************************/
 
+/*! \brief Block Transfer Type
+*
+*    This structure holds the block mode, crc, and crc_value.
```

Comment:
If there must be a function brief try to make it add value. In what context is this structure used? otherwise I would argue for dropping the comment block

Reply:
done

Reply:
Thanks!